spec/sections/usage.adoc

```diff
@@ -17,10 +17,11 @@ MUST follow Bash expansion conventions.
 The following minimum variables MUST be provided:
 
 * JOB_OUTPUT_DIR: Absolute path where all output products MUST be placed by job for Seed executor capture.
```

Comment:
I was thinking we should change this to "OUTPUT_DIR" or "SEED_OUTPUT_DIR". Thoughts?